### ISSUE TITLE
Swap out link options with anchor_link module version

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa#1412]
The built in CKEditor link options are not as robust as Drupal 7, even with the addition of the editor_advanced_link module. 

It looks like the anchor_link module (https://www.drupal.org/project/anchor_link) may provide additional options to allow selection of different types of links.

# Steps
- [ ] Investigate whether anchor_link is a drop in replacement for link with no loss of functionality and is compatible with linkit and editor_advanced_link.
- [ ] Get consensus from UIOWA Drupal devs about this approach.
- [ ] Add module to composer.json and composer.lock.
- [ ] Update sitenow profile config for any text formats that allow the link button to remove existing link buttons (add, remove) and add the anchor_link versions of them.

**Comment by joewhitsitt** [comment]
SiteNow is using LinkIt for our link buttons. I haven't looked at this, but I'd be open to the anchor button, but I think we want to maintain internal path linking through LinkIt until something better comes along.

**Comment by pyrello** [comment]
@joewhitsitt Good call! I updated the steps to include compatibility with Linkit in the investigation.

**Comment by joewhitsitt** [comment]
https://www.drupal.org/project/linkit/issues/3022261

**Comment by joewhitsitt** [comment]
Going back to the anchor_link module. I remembered this yesterday when talking with Makur.

This module/button only finds anchors in the current wysiwyg field. not across multiple areas (e.g. paragraphs, layout blocks) in the page.
https://www.drupal.org/project/anchor_link/issues/3144982

**Comment by joewhitsitt** [comment]
I propose using anchor_link but only implementing the "anchor" button for now.

**Comment by joewhitsitt** [comment]
This requires https://ckeditor.com/cke4/addon/link and https://ckeditor.com/cke4/addon/fakeobjects in the libraries folder and I don't think there is a composer/bower option...

**Comment by pyrello** [comment]
Closing this in favor of the epic: #3311
